module/Application/Module.php

```diff
@@ -9,6 +9,12 @@
 
 namespace Application;
 
+use Application\View\Helper\Acl;
+use Application\View\Helper\FileUrl;
+use Application\View\Helper\Infima;
+use Application\View\Helper\JobCategories;
+use Application\View\Helper\ModuleIsActive;
+use Application\View\Helper\ScriptUrl;
```

Comment:
I'm not really a fan of adding all these imports when each class is only used once here. In my opinion it even makes the code slightly less readable.

Reply:
I personally think it makes it more readable. But I guess it's rather subjective.